shared/src/persistence/dynamo/users/createMigratedPetitionerUser.js

```diff
@@ -65,6 +66,34 @@ exports.createMigratedPetitionerUser = async ({ applicationContext, user }) => {
       userId: response.User.Username,
     });
   } catch (err) {
+    // the user already exists in Cognito
     applicationContext.logger.error(err);
+    const response = await applicationContext
```

Comment:
reusing existing pattern for trying to get the user from Cognito if creating the user in Cognito fails. Then proceed with creating the DynamoDB record...